terraform/ec2/common/linux/output.tf

```diff
@@ -17,3 +17,6 @@ output "proxy_instance_proxy_ip" {
   value = module.proxy_instance.proxy_ip
 }
 
+output "cwa_onprem_assumed_iam_role_arm" {
```

Comment:
nitpick - I think there's a typo in `arm`. I'm guessing this is supposed to be `arn`

Reply:
Would need to change it in common https://github.com/aws/amazon-cloudwatch-agent-test/blob/1eb7bb1de72d9b765891906f08d23ffafe8ff927/terraform/common/output.tf#L12 as well.